tests/integrationv2/fixtures.py

```diff
@@ -36,3 +47,42 @@ def _fn(provider_class: Provider, options: ProviderOptions, timeout=5):
         # Whether the processes succeeded or not, clean then up.
         for p in processes:
             p.join()
+
+
+def _swap_mtu(device, new_mtu):
+    """
+    Swap the device's current MTU for the requested MTU.
+    Return the original MTU so it can be reset later.
+    """
+    cmd = ["ifconfig", device]
+    p = subprocess.Popen(cmd, stdin=subprocess.PIPE, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    mtu = 65536
+    for line in range(0, 5):
+        output = p.stdout.readline().decode("utf-8")
+        if ("MTU:" in output):
+            word_list = output.split()
+            mtu_list = word_list[3].split(':')
+            mtu = mtu_list[1]
+            break
+
+    p.wait()
+
+    subprocess.call(["ifconfig", device, "mtu", str(new_mtu)])
```

Comment:
The suggested alternative is ` ip link set dev <INTERFACE> mtu 1400`, but again, a nit as not all distros are handling this consistently.

Reply:
As above, tested on 16.04 and 18.04. I'm leaning heavily on our apt-get dependency system. Once we support multiple distros, verifying `ip link` will be one of several things we need to do.